### PR TITLE
add bioservices

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hadim** [comment]
Ping @conda-forge/bioconda-recipes about this package. Are you ok adding it to conda-forge?

**Comment by hadim** [comment]
@conda-forge/staged-recipes please review

File: recipes/bioservices/meta.yaml
Comment (jakirkham) [inline recipes/bioservices/meta.yaml:49]: Is it only or does it support newer versions of the license as well?

**Comment by jakirkham** [comment]
LGTM. Had one question above

File: recipes/bioservices/meta.yaml
Comment (hadim) [reply recipes/bioservices/meta.yaml:49]: How could I check that? See https://github.com/cokelaer/bioservices/blob/master/LICENSE

Comment (jakirkham) [reply recipes/bioservices/meta.yaml:49]: How about this?

https://github.com/cokelaer/bioservices/blob/a334adaf6ea6ea87687517bfab0b0c090b68fcca/LICENSE#L570-L594

Comment (hadim) [reply recipes/bioservices/meta.yaml]: Thanks for the catch!

**Comment by jakirkham** [comment]
Just noticed they also mention BSD-3-Clause despite including GPL 3. Asking about that here ( https://github.com/cokelaer/bioservices/issues/187 )

**Comment by hadim** [comment]
> Just noticed they also mention BSD-3-Clause despite including GPL 3. Asking about that here ( [cokelaer/bioservices#187](https://github.com/cokelaer/bioservices/issues/187) )

Thanks for doing that.

**Comment by hadim** [comment]
Is that ok to merge here and fix the license once that has been clarified? Or is it a blocker in your opinion @jakirkham ?

**Comment by jakirkham** [comment]
I've tried to poke that issue. It feels a little messy atm since there is a wide gap between GPL-3 and BSD-3. I think just getting an answer from the author would help

**Comment by jakirkham** [comment]
cc @conda-forge/bioconda-recipes (as it looks like the same recipe may be in Bioconda)

**Comment by hadim** [comment]
The licensing seems to have been clarified. Is that ok to merge @jakirkham or do we need specific approval from bioconda folks?

**Comment by jakirkham** [comment]
We need feedback from Bioconda since they already package this

**Comment by mbargull** [comment]
Hi @hadim and @jakirkham, sorry for the late reply.
The license should be `GPL-3.0-only` since the source does not seem to have the "or any later version" phrase in it (apart from the GPL text itself which just describes on how to apply it).
What is the intend on moving this over from Bioconda?

Cheers,
Marcel
(I'm going AFK for today, but will get back to you in the following days.)

**Comment by hadim** [comment]
@mbargull the main motivation for moving it on conda-forge is to rely only on the conda-forge channel to build our stack. As adding more channels tends to the complexity the graph of deps solving step with conda/mamba.

**Comment by bgruening** [comment]
@hadim thats ok. Just make sure you remove the recipe from the bioconda side.

**Comment by hadim** [comment]
Is that ok to merge here?

@bgruening one merged here I can remove the recipe from bioconda. Is there is a specific procedure to follow?

**Review by mbargull — APPROVED** [review]

**Comment by mbargull** [comment]
Sorry for the delay. 
> As adding more channels tends to the complexity the graph of deps solving step with conda/mamba.

Understandable, both implementations have different deficiencies with this, indeed.

> @bgruening once merged here I can remove the recipe from bioconda. Is there is a specific procedure to follow?

Yes, please open a PR on bioconda-recipes to remove the recipe with a reference to this PR; much appreciated!

**Comment by hadim** [comment]
Thank you. See the PR https://github.com/bioconda/bioconda-recipes/pull/28010

**Comment by cokelaer** [comment]
@hadim @bgruening 
I'm the main author of bioservices and I am not sure I understood the rational in moving bioservices from bioconda to conda-forge. BioServices is indeed a software that is dedicated to bioinformatics and biological web services access and I have the feeling it would be more visible in bioconda rather than conda-forge ? For end-users, it should be equivalent of course but I ust want to make sure that it makes sense before updating the documentation. thanks

**Comment by hadim** [comment]
@cokelaer I think the main motivation is to reduce the number of conda channels in a custom conda environment. Having a lower number of channels reduces the dep graph solver computational time and in consequence reduces the conda env creation in CI, local machines, docker images, production, etc.

Since (if I am not mistaken), bioconda is relying on conda-forge as you said it should not change anything to the main user. 

As for the visibility I understand having it in conda-forge makes it less visible but maybe this is not the most important channel of communication?

I apologize if you feel strongly against the move.

**Comment by cokelaer** [comment]
@hadim thanks for the quick reply and explanations.  If it does not change anything for the user, I believe this move is for the best. thanks for your work. best

**Pull request merged**
